components/scientific/pari/Makefile

```diff
@@ -22,7 +22,7 @@ include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=		pari
 COMPONENT_VERSION=	2.17.0
-COMPONENT_REVISION=	1
+COMPONENT_REVISION=	2
 COMPONENT_SUMMARY=	The PARI Computer Algebra System
 COMPONENT_PROJECT_URL=	https://pari.math.u-bordeaux.fr
 COMPONENT_SRC=		pari-$(COMPONENT_VERSION)
```

Comment:
If pari-elldata is being used for the test you need to add

# Manually added dependencies
TEST_REQUIRED_PACKAGES += library/math/pari-elldata

With TEST_REQUIRED_PACKAGES our build system makes sure to have it installed when gmake test is run.